### ISSUE TITLE
link to config

### ISSUE THREAD

**Issue by agentlame** [creesch/reddit-moderator-toolbox#706]
Like link to settings, but for TB config.

Example: https://www.reddit.com/r/toolbox#?tbconfig=mmacros&setting=newmacro

**Comment by creesch** [comment]
Yup, just the tabs would even work I guess. Config isn't as complex as individual settings.

**Comment by agentlame** [comment]
You're right.
